datacatalog/pkg/manager/impl/artifact_manager.go

```diff
@@ -369,6 +379,10 @@ func (m *artifactManager) UpdateArtifact(ctx context.Context, request *datacatal
 
 	// update artifact in DB, also replaces/upserts associated artifact data
 	artifactModel.ArtifactData = artifactDataModels
+	logger.Infof(ctx, "ArtifactModel is %+v", artifactModel)
+	logger.Infof(ctx, "ArtifactModel ArtifactData is %+v", artifactModel.ArtifactData)
+	logger.Infof(ctx, "ArtifactModel SerializedMetadata is %+v", artifactModel.SerializedMetadata)
```

Comment:
Is this logging necessary or debugging information? If anything it should be `Debugf`, but I would be OK removing as well.

Reply:
thanks for suggestions, i have changed it to Debugf for debug purpose.